src/bin/pgcopydb/parsing_utils.c

```diff
@@ -1080,3 +1080,12 @@ freeKeyVal(KeyVal *parameters)
 
 	parameters->count = 0;
 }
+
+/*
+ * returns true, if the string contains "warning"
+ */
+bool
+isWarning(const char* str)
+{
+	return strstr(str, "warning") != NULL;
```

Comment:
What if the buffer contains the user-provided string "warning", as in a column name or a table name for instance?

Reply:
yeah, I had thought of it too. What do you think is the best way to filter warnings out?

How about this -

"**pg_dump: warning:**" and "**pg_restore: warning:**"

Reply:
Looks better already. I suppose it needs to be matched at the start of a line only?